packages/dso-toolkit/src/components/logo/logo.args.ts

```diff
@@ -13,16 +17,23 @@ export const logoArgTypes: ArgTypes<LogoArgs> = {
       type: "text",
     },
   },
+  labelUrl: noControl,
   ribbon: {
     control: {
       type: "text",
     },
   },
+  dsoClick: {
+    ...noControl,
+    action: "dsoClick",
+  },
 };
 
 export function logoArgsMapper(a: LogoArgs): Logo {
   return {
     label: a.label,
+    labelUrl: a.labelUrl,
     ribbon: a.ribbon,
+    dsoClick: (e) => a.dsoClick(e.detail),
```

Comment:
`logo.args.js:24 Uncaught TypeError: a.dsoClick is not a function
    at o.dsoClick (logo.args.js:24:1)
    at EventPart.handleEvent (lit-html.ts:2068:1)
    at D (index.js:604:7)
    at Object.emit (index.js:593:14)
    at HTMLAnchorElement.handleClick (logo.tsx:44:19)`

Reply:
Zou je me hierbij kunnen helpen, @tfrijsewijk?